src/fidesops/core/config.py

```diff
@@ -73,12 +74,7 @@ def assemble_connection_url(
             # If the whole URL is provided via the config, preference that
             return v
 
-        return "redis://:{password}@{host}:{port}/{db_index}".format(
-            password=values["PASSWORD"],
-            host=values["HOST"],
-            port=values["PORT"],
-            db_index=values.get("DB_INDEX"),
-        )
+        return f"redis://{values.get('USER', '')}:{values['PASSWORD']}@{values['HOST']}:{values['PORT']}/{values.get('DB_INDEX')}"
```

Comment:
If `DB_INDEX` is `NONE` the string will be `redis://{some url}/None` so maybe default to `''` if `DB_INDEX` is `None`.

```suggestion
        return f"redis://{values.get('USER', '')}:{values['PASSWORD']}@{values['HOST']}:{values['PORT']}/{values.get('DB_INDEX', '')}"
```

Reply:
Good spot, thanks!